contrib/debian/netdata-plugin-apps.postinst

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+case "$1" in
+  configure|reconfigure)
+    setcap "cap_dac_read_search=eip cap_sys_ptrace=eip" /usr/libexec/netdata/plugins.d/apps.plugin
```

Comment:
Is this `setcap` enough to access all `/proc` files? Are we not missing the `chmo 4750` here?

Reply:
It’s consistent with what we have been setting already, and that seems to work correctly. `CAP_DAC_READ_SEARCH` allows reading any file (it’s mostly intended for backup tools, but is also useful for this type of thing), and `CAP_SYS_PTRACE` bypasses the few non-DAC access checks on files the apps plugin is supposed to be reading.

If we’re setting capabilities correctly, we should not need the SUID bit set (that’s actually the point of file capabilities like this).

Reply:
Thank you!